### ISSUE TITLE
[1.38.0-beta.1] Bold and Italics buttons in dismiss issue comments missing visual indicator of pressed state

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1470]
When you click or press with a keyboard the Bold or Italic setting, there is no visual indicator that it's currently on. You have to type to see if it worked.
<img width="403" height="281" alt="Image" src="https://github.com/user-attachments/assets/440a23bb-02cd-4eb7-a509-71dbe0045896" />

Add some sort of CSS styles to make it look depressed or activated.

**Comment by pattonwebz** [comment]
Button state is not handling aria-pressed and visually indicates the states.

https://github.com/user-attachments/assets/768093d3-67a8-43e6-a2ae-8373446c3036
